capella_console_client/repeat_requests.py

```diff
@@ -0,0 +1,145 @@
+from typing import Optional, Dict, Any, List, Union
+from datetime import datetime, timedelta
+
+import geojson
+from dateutil.parser import parse, ParserError
+
+from capella_console_client.session import CapellaConsoleSession
+from capella_console_client.validate import _snake_to_camel
+from capella_console_client.enumerations import (
+    ObservationDirection,
+    OrbitState,
+    ProductType,
+    ProductClass,
+    OrbitalPlane,
+    RepeatCollectionTier,
+    InstrumentMode,
+    Polarization,
+    ArchiveHoldback,
+)
+
+COLLECT_CONSTRAINTS_KEYS = set(
```

Comment:
I'd rename to `REPEAT_REQUEST_COLLECT_CONSTRAINTS_KEYS` plus make this a `frozenset`

Reply:
Done, moved this along with the repetition keys.